File: CPP/Disk.cpp
class Disk
{
    private:
        /* data */
        string type;
        string capacity;
        int price;

    public:
        Disk(/* args */) {
        }
        Disk(string d, string m, int y) {
            this->type = d;
            this->capacity = m;
            this->price = y;
        }

        void setType(string d) { this->type = d; }
        void setCapacity(string m) { this->capacity = m; }
        void setPrice(int y) { this->price = y; }

        string getType() { return this->type; }
        string getCapacity() { return this->capacity; }
        int getPrice() { return this->price; }

        void printDetDisk() {
            cout << "----------------------" << endl;
            cout << "Disk Type      : " << this->type << endl;
            cout << "Disk Capacity  : " << this->capacity << endl;
            cout << "Price          : " << this->price << endl;
        }

        ~Disk() {
        }
};

File: PHP/Disk.php
<?php

class Disk{
    private $type;
    private $capacity;
    private $price;

    public function __construct($t, $c, $p){
        $this->type = $t;
        $this->capacity = $c;
        $this->price = $p;
    }

    public function setType($t){
        $this->type = $t;
    }
    public function getType(){
        return $this->type;
    }

    public function setCapacity($c){
        $this->capacity = $c;
    }
    public function getCapacity(){
        return $this->capacity;
    }

    public function setPrice($p){
        $this->price = $p;
    }
    public function getPrice(){
        return $this->price;
    }

    public function printDisk()
    {
        echo "---------------------- <br>";
        echo "Disk Type      : ". $this->type ."<br>";
        echo "Disk Capacity  : ". $this->capacity ."<br>";
        echo "Price          : ". $this->price ."<br>";
    }

    public function __destruct(){ }
}

?>